### PR TITLE
#1793 Make dropdown for filter and search bar narrow

### LINKED ISSUE #1793: Make dropdown for filter and search bar narrow
So pagination is on the same line

image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇  _ Do a quick check before the merge.

### PR type
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?
- [x] PR closes #1793
- [x] issue if because of buefy classes being added (flex-wrap) so I enabled it only on phone using css

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main-nuxt** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### Had issue bounty label ?
- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation
- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [ ] My fix has changed **something** on UI, a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by yangwao** [comment]
I would put filter section to 1/2 or 3/4 of the current size in a small viewport

This is on collection detail tho https://deploy-preview-1813--koda-nuxt.netlify.app/rmrk/collection/086b3cff289d887136-QIAN?tab=collection
image

On collections let's make the same adjustments, though it's a shared component?
image


Otherwise seems good to me!

**Comment by yangwao** [comment]
> To get started, I would add a fixed width of 200px on `<select>` dropdown
> Later we could move the pagination result (1 - 9 / 4935) outside the searchBar

That's way to go

**Comment by prachi00** [comment]
@yangwao @roiLeo fixed the width to `200px` on smaller desktop screens

**Comment by yangwao** [comment]
Perfect, I’ve sent the payout https://kusama.subscan.io/extrinsic/0x7fcd3fc25de56cb077880b8ca1fdee93631ba99fa8ddd54453cd95077ea798cc

Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

File: components/rmrk/Gallery/Search/SearchSortDropdown.vue [hunk 1]
```diff
@@ -37,3 +38,11 @@ export default class SearchSortDropdown extends Vue {
   }
 }
 </script>
+
+<style scoped lang="scss">
+ .select-dropdown {
+  @media screen and (max-width: 1216px) and (min-width: 768px) {
+    width: 200px;
+  }
```
Comment (roiLeo) [inline components/rmrk/Gallery/Search/SearchSortDropdown.vue:46]: I would put the width on all screen

File: components/rmrk/Gallery/Search/SearchBarCollection.vue [hunk 2]
```diff
@@ -148,3 +148,14 @@ export default class SearchBar extends Vue {
...
+
+<style lang="scss">
+.field-group-container {
+  .is-grouped-multiline {
+    flex-wrap: initial!important;
+    @media screen and (max-width: 768px) {
+      flex-wrap: wrap!important;;
+    }
+  }
+}
```
Comment (roiLeo) [inline components/rmrk/Gallery/Search/SearchBarCollection.vue:160]: Oh my, what does this fix?

File: components/rmrk/Gallery/Search/SearchSortDropdown.vue (hunk 1 above)
Comment (prachi00) [reply components/rmrk/Gallery/Search/SearchSortDropdown.vue:46]: @roiLeo on larger ones , the search bar was looking really big tho

File: components/rmrk/Gallery/Search/SearchBarCollection.vue (hunk 2 above)
Comment (prachi00) [reply components/rmrk/Gallery/Search/SearchBarCollection.vue:160]: @roiLeo due to the `is-group-multiline` class from buefy, it was being applied for desktop also, so` flex-wrap` was messing things on smaller desktop screens, so I applied flex-wrap only on mobile screens and scoped it through the parent class `field-group-container`

File: components/rmrk/Gallery/Search/SearchSortDropdown.vue (hunk 1 above)
Comment (roiLeo) [reply components/rmrk/Gallery/Search/SearchSortDropdown.vue:46]: Alright, then we could try to fix a width to input too as we don't need to search super long query

**Pull request merged**